src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -612,18 +601,23 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
-  public void verifyExpandedAdVisibleInSlot(String slotSelector, WebElement slot) {
-    waitForSlotExpanded(slot);
+  private void triggerBLB() {
+    scrollToFooter();
 
-    boolean adVisible = new AdsComparison().isAdVisible(slot, slotSelector, driver);
+    simulateUserActivity(Duration.ofSeconds(3));
+  }
 
-    extractGptInfo(slotSelector);
+  private void simulateUserActivity(Duration duration) {
+    simulateUserActivity(duration, Duration.ofMillis(500));
+  }
 
-    if (!adVisible) {
-      throw new WebDriverException("Ad is not present in " + slotSelector);
+  private void simulateUserActivity(Duration duration, Duration waitDuration) {
```

Comment:
I wish we know why BLB behaves like that, so we don't need to have this method... but we can investigate it outside scope of this task.